### DISCUSSION TITLE
Working on Gitlab to Github migration

### DISCUSSION THREAD

**Discussion by syedobaid123** [community/community#112399] [Actions]
### Select Topic Area

General

### Body

I am working on Gitlab to Github migration for users, repos and pipelines. Maybe users I will migrate manually, repositories I am planning to use github repository importer and for pipelines, I am planning to use Github Actions importer.

I want to know is there any license required to use github actions importer and github repository importer. 

Kindly let me know

**Comment by Abdullahxz** [comment]
I don't think you need any license for this.

Source: [github-actions-importer legal notice](https://docs.github.com/en/actions/migrating-to-github-actions/automated-migrations/automating-migration-with-github-actions-importer#legal-notice)
